claimManagement/src/main/java/org/openimis/imisclaims/Escape.java

```diff
@@ -0,0 +1,15 @@
+package org.openimis.imisclaims;
+
+public class Escape {
+
+    public boolean CheckCHFID(){
```

Comment:
Make this method as generic as in Policies App with a string parameter as InsureeNumber. Here is the actual validation.